src/MassTransit/Transports/ReceiveEndpointLoggingExtensions.cs

```diff
@@ -61,20 +61,17 @@ public static void LogFaulted<T>(this ConsumeContext<T> context, TimeSpan durati
         {
             if (_messages.IsErrorEnabled)
             {
-                var faultMessage = GetFaultMessage(exception);
-
                 _messages.Error(
-                    $"R-FAULT {context.ReceiveContext.InputAddress} {GetMessageId(context.ReceiveContext)} {TypeMetadataCache<T>.ShortName} {consumerType}({duration}) {faultMessage}");

```

Comment:
I would really like to see the `FaultMessage` retained in the message, since it's easily searchable through other log writers. If you could add it back, but still call the separate method, that would be awesome and I'll pull this into develop.